base/db/R/query.trait.data.R

```diff
@@ -6,417 +6,6 @@
-  } else {
-    data <- NULL
-  }
-  return(data)
-}
-##==================================================================================================#
-
-
-##--------------------------------------------------------------------------------------------------#
-##' renames the variables within output data frame trait.data
-##'
-##' @param data data frame to with variables to rename
-##'
-##' @seealso used with \code{\link[PEcAn.MA]{jagify}};
-##' @export
-##' @author David LeBauer
-rename_jags_columns <- function(data) {
-
-  # Change variable names and calculate obs.prec within data frame
-  transformed <-  transform(data,
-                            Y        = mean,
-                            se       = stat,
-                            obs.prec = 1 / (sqrt(n) * stat) ^2,
-                            trt      = trt_id,
-                            site     = site_id,
-                            cite     = citation_id,
-                            ghs      = greenhouse)
-
-                                        # Subset data frame
-  selected <- subset(transformed, select = c('Y', 'n', 'site', 'trt', 'ghs', 'obs.prec',
-                                    'se', 'cite'))
-                                        # Return subset data frame
-  return(selected)
-}
-##==================================================================================================#
-
-
-
-##--------------------------------------------------------------------------------------------------#
-##' Change treatments to sequential integers
-##'
-##' Assigns all control treatments the same value, then assigns unique treatments
-##' within each site. Each site is required to have a control treatment.
-##' The algorithm (incorrectly) assumes that each site has a unique set of experimental
-##' treatments.
-##' @name assign.treatments
-##' @title assign.treatments
-##' @param data input data
-##' @return dataframe with sequential treatments
-##' @export
-##' @author David LeBauer, Carl Davidson, Alexey Shiklomanov
-assign.treatments <- function(data){
-  data$trt_id[which(data$control == 1)] <- "control"
-  sites <- unique(data$site_id)
-  # Site IDs may be returned as `integer64`, which the `for` loop
-  # type-coerces to regular integer, which turns it into gibberish.
-  # Looping over the index instead prevents this type coercion.
-  for (si in seq_along(sites)) {
-    ss <- sites[[si]]
-    site.i <- data$site_id == ss
-    #if only one treatment, it's control
-    if (length(unique(data$trt_id[site.i])) == 1) data$trt_id[site.i] <- "control"
-    if (!"control" %in% data$trt_id[site.i]){
-      PEcAn.logger::logger.severe(paste0(
-        "No control treatment set for site_id ", unique(data$site_id[site.i]),
-        " and citation id ", unique(data$citation_id[site.i]), ".\n",
-        "Please set control treatment for this site / citation in database.\n"
-      ))
-    }
-  }
-  return(data)
-}
-
-drop.columns <- function(data, columns){
```

Comment:
I don't see `drop.columns` in any of the added files. Accidental removal, or am I missing it?

...Hmm, but it also seems unused, so maybe this is / should be an intentional removal?

Reply:
It was an accidental omission, but I searched the pecan code and I don't think it's used anywhere, so I think I may just leave it out.